answers.md

```diff
@@ -0,0 +1,8 @@
+1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+describe() is a container for related tests. In this challenge one describe is used for all GET endpoint testing and one is used for all POST endpoint testing. it() is for individual tests within a describe().
```

Comment:
One important thing to note about `describe`s is that you can actually nest them. That means this sort of thing would work:
```javascript
describe('GET endpoints', () => {
  describe('/', () => {
    it('should return OK status', async () => {
      const response = await request(server).get('/');
      expect(response.status).toBe(200);
    });
  });

  describe('/data', () => {
    it('should return OK status', async () => {
      const response = await request(server).get('/data');
      expect(response.status).toBe(200);
    });
  });
});
```

Reply:
These similar tests could also be put into a helper function in order to reduce repetition:
```javascript
const testOkFor = endpoint => async () => {
  const response = await request(server).get(endpoint);
  expect(response.status).toBe(200);
};

describe('GET endpoints', () => {
  describe('/', () => {
    it('should return OK status', testOkFor('/'));
  });

  describe('/data', () => {
    it('should return OK status', testOkFor('/data'));
  });
});
```